cloudify_cli/bootstrap/bootstrap.py

```diff
@@ -262,15 +288,16 @@ def get_protocol(rest_port):
                 MANAGER_USER_RUNTIME_PROPERTY]
         manager_key_path = manager_node_instance.runtime_properties[
             MANAGER_KEY_PATH_RUNTIME_PROPERTY]
-        rest_port = \
-            manager_node_instance.runtime_properties[REST_PORT]
-        protocol = get_protocol(rest_port)
     else:
         manager_ip = env.outputs()['manager_ip']
         manager_user = manager_node.properties['ssh_user']
         manager_key_path = manager_node.properties['ssh_key_filename']
-        rest_port = manager_node_instance.runtime_properties[REST_PORT]
-        protocol = get_protocol(rest_port)
+        # security_config = manager_node.properties['security']
```

Comment:
why are these commented out? is it part of the old path? if so, maybe simply remove them?